tools/cldr-code/src/main/java/org/unicode/cldr/util/SupplementalCalendarData.java

```diff
@@ -0,0 +1,229 @@
+package org.unicode.cldr.util;
+
+import com.ibm.icu.impl.locale.XCldrStub.ImmutableMap;
```

Comment:
This is wrong; we should never have XCldrStub in any CLDR code.

Reply:
done